### ISSUE TITLE
Pattern Matching of Test Not Working -> Error: No tests found

### ISSUE THREAD

**Issue by magbeat** [mbolotov/intellij-playwright#25]
Hi there,

I am trying to run a specific test.

When I run the test, it cannot find the test using the provided pattern.

![image](https://user-images.githubusercontent.com/1089410/221624485-4bba5144-f514-49b5-8f0e-6d1ab1b12ae4.png)

Result:

![image](https://user-images.githubusercontent.com/1089410/221624638-33f42c7a-b752-440d-84f9-9e6ebd83f198.png)

If I copy over the command to a terminal and change it to something like this:

`/var/lib/snapd/snap/bin/npx playwright test --reporter /tmp/intellij-playwright-reporter3214380591270717897.js tests/example.spec.ts -g get started link$` 

it works.

OS: Linux (Fedora)
IDE: Webstorm 2022.3.2
Plugin Version: 1.3.1

**Comment by mbolotov** [comment]
Hi Markus, 
I'm unable to reproduce the issue. 
Could you provide a sample project to reproduce it?

**Comment by sterschl** [comment]
I have the same problem when the test name contains spaces:

```
test('should test this page', async () => {
...
}
```

The command to execute the test is the following:

`playwright test --headed --project=chromium --reporter intellij-playwright-reporter11841165207304899153.js tests/example.spec.ts -g "^(.*[\\\/])?example.spec.ts.should test this page$"`

**Comment by oantoshchenko** [comment]
Have the same issue. I couldn't figure out a way to run any test. I tried running from the margin button from the project structure (directory context menu > run > Playwright). Environment:
- MacOS 13.3.1 
- WebStorm 2023.1.1
- `playwright.config.ts`:

```ts
import type { PlaywrightTestConfig } from '@playwright/test';

const config: PlaywrightTestConfig = {
	webServer: {
		command: 'npm run build && npm run preview',
		port: 4173
	},
	testDir: 'e2e-tests'
};

export default config;
```

- I also tried changing the name of the test file to `test.ts` and giving the test itself name `test`: 
```ts
test('test', async ({ page }) => {
```

**Comment by mbolotov** [comment]
hi folks, 
could anyone provide a sample project to reproduce the problem?
I still can't do that.

**Comment by oantoshchenko** [comment]
Dude, you are charging money for this product, and expecting your customers to also work on it?

**Comment by mbolotov** [comment]
> Dude, you are charging money for this product, and expecting your customers to also work on it?

Yes, I am. I'm expecting that my customers are interested in fixing their problems.

**Comment by yparenteau** [comment]
Same issue here "No tests found" while trying the run the test examples from the default Playwright installation.
`yarn create playwright`

/Users/xyz/Projects/poc/playwright/node_modules/.bin/playwright test --reporter /var/folders/m3/fhnlfkp91sj648w3_8m0ng980000gn/T/intellij-playwright-reporter6414323199617572113.js tests/demo-todo-app.spec.ts -g '^(.*[\\\/])?demo-todo-app.spec.ts.New Todo'

**Comment by sw-tracker** [comment]
@mbolotov we are having the same issue as the others are reporting.

Would you be able to provide an example project for us that works? Then we can compare we our setup with your working setup. Please also include the intellij run config in your example repository and some docs.

Thanks!

**Comment by mbolotov** [comment]
Can anyone try this bundle with fix?
[Maestro-1.3.2-SNAPSHOT.zip](https://github.com/mbolotov/intellij-playwright/files/11747376/Maestro-1.3.2-SNAPSHOT.zip)
I've relaxed the pattern for selecting the test. 
I wonder if this template does not select a wider scope than expected

**Comment by mbolotov** [comment]
fixed in `1.3.2` version
